plugins/inara.py

```diff
@@ -74,6 +83,50 @@ def _(x):
 STATION_UNDOCKED: str = '×'  # "Station" name to display when not docked = U+00D7
 
 
+class Credentials(NamedTuple):
+    """
+    Credentials holds an inara API payload
```

Comment:
That docstring doesn't seem strictly true.  This is the identifier for a queue that *holds* payloads.